config.mk

```diff
@@ -0,0 +1,13 @@
+#####################################
```

Comment:
this file is auto generated, so it should not be in this commit, but in the .gitignore